### PR TITLE
Fix: Resolve name inconsistencies of GLA Demolition Ability for all latin languages

### PR DESCRIPTION
**Pull request by xezon**
This change fixes name inconsistencies of the GLA Demolition Ability for all latin languages.

### CODE REVIEW

**Comment by xezon** [comment]
Now also added correct names to HELP:Demolitions-01. Documentation updated. Looks good.

File: Patch104pZH/GameFilesEdited/Data/generals.str [hunk 1]
```diff
@@ -45952,15 +45952,17 @@ RU: "HINT:\nAdvanced Traps are more powerful than normal Demo Traps"
...
 
+// Patch104p @fix xezon 20/08/2023 Fix name discrepancies with CONTROLBAR:UpgradeSuicideBomb for all latin languages. (#2250)
+
 HELP:Demolitions-01
-US: "HINT:\nThe Demolitions upgrade gives units the ability to detonate"
-DE: "TIPP:\nDas TNT-Upgrade ermöglicht Ihren Einheiten, sich selbst zu sprengen."
-FR: "ASTUCE :\nL'amélioration Charges explosives confère à vos unités la capacité d'exploser."
-ES: "PISTA:\nLa mejora de demoliciones permite que las unidades detonen"
-IT: "SUGGERIMENTO:\nIl potenziamento Demolizione consente alle\nunità di esplodere."
+US: "HINT:\nThe Demolitions ability enables all units to detonate"
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/generals.str:45958]: Wasn't the previous correct? Demolitions upgrade gives units the Suicide ability.

There is no "Demolitions ability".

File: Patch104pZH/GameFilesEdited/Data/generals.str [hunk 2]
```diff
@@ -90233,16 +90236,18 @@ RU: "Штурмовой бронетранспортёр\nШтурмовая У
...
-DE: "Verbesserte Sprengfalle\nTNT-Upgrade\nMehr Schaden durch Bomben\nBeginnt mit Sprengfallen\nKampfbikes tragen Bomben\nKeine getarnten Einheiten\nBomben-LKWs kosten weniger"
-FR: "Pièges explosifs C-404\nAmélioration Explosifs pour toutes les unités\nLes Terroristes font plus de dégâts\nCommence avec des mines artisanales\nMotos de combat pourvues au départ d'un Terroriste\nPas d'unités camouflées\nPotentiel chimique nul\nCoût réduit des camions piégés"
-ES: "Trampa de demolición avanzada\nEmpieza con la mejora de Demolición\nLos terroristas hacen más daño\nEmpieza con trampa explosiva\nLas motos de combate empiezan con terrorista\nSin unidades camufladas\nSin capacidad de toxinas\nLos camiones bomba cuestan menos"
-IT: "- Trappola demolitrice avanzata\n- Inizia con agg. Demolizione\n- Terroristi più efficaci\n- Inizia con Ordigno esplosivo\n- Moto da combattimento guidate da un Terrorista\n- Nessuna unità mimetizzata\n- Nessuna tossina\n- Camion-bomba economici"
+US: "Advanced Demo Trap\nDemolitions Ability\nTerrorists do more damage\nStarts with Booby Trap\nCombat Cycles begin with Terrorist\nNo camouflaged units\nBomb Trucks cost less"
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/generals.str:90242]: "Demolitions upgrade"

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/generals.str:90242]: It is called "Demolitions Ability" in the tool tip, so I went with that.

**Pull request merged**